<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>itemForm</class>
 <widget class="QMainWindow" name="itemForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>46</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(197, 197, 197);

hover{
	background-color: rgb(114, 236, 255)
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>ID</string>
      </property>
      <property name="margin">
       <number>5</number>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="idLable">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(216, 255, 249);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="margin">
       <number>5</number>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Age</string>
      </property>
      <property name="margin">
       <number>5</number>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="ageLable">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(216, 255, 249);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="margin">
       <number>5</number>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Name</string>
      </property>
      <property name="margin">
       <number>5</number>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="nameLable">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(216, 255, 249);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="margin">
       <number>5</number>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Address</string>
      </property>
      <property name="margin">
       <number>5</number>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="addressLable">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(216, 255, 249);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="margin">
       <number>5</number>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Salary</string>
      </property>
      <property name="margin">
       <number>5</number>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="salaryLable">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(216, 255, 249);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="margin">
       <number>5</number>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
